src/main/java/com/telefonica/iot/cygnus/sinks/OrionHDFSSink.java

```diff
@@ -575,14 +588,14 @@ public void aggregate(CygnusEvent cygnusEvent) throws Exception {
                 
                 // create a line and aggregate it
                 String line = "{"
-                    + "\"" + Utils.encodeHive(Constants.RECV_TIME_TS) + "\":\"" + recvTimeTs / 1000 + "\","
-                    + "\"" + Utils.encodeHive(Constants.RECV_TIME) + "\":\"" + recvTime + "\","
+                    + "\"" + Utils.encodeHive(Constants.RECV_TIME_TS) + "\":\"" + recvTimeTs / 1000 + "\","  
```

Comment:
This code belongs to json method, but I had to change the csv method. The changes are spaces which doesn't affect to the main code.